project_configs/test-extract-uclh-omop-cdm.yaml

```
@@ -0,0 +1,23 @@
+#  Copyright (c) 2022 University College London Hospitals NHS Foundation Trust
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+project:
+    name: "myproject"
```

Why don't we make this be for the 100 days project. Can rename the project `uclh-nasogastric-tube-project-ngt-only-full-dataset.yaml` or `100-days.yaml`

```suggestion
    name: "uclh-nasogastric-tube-project-ngt-only-full-dataset"
```

---

It should actually match the name of the file too. The slug is not in our control AFAIK, it depends on what we get in the json from OMOP-ES - is your code suggestion that name (I honestly have no idea how to check)?

---

Yep it is, and I think that'd be part of the project setup in agreeing on the name. Have you got your setup on GAE05?

---

they do not